mesonbuild/interpreter.py

```diff
@@ -1943,17 +1943,15 @@ def get_cross_property_method(self, args, kwargs):
                                    'static',
                                    'version',
                                    'private_headers',
-                                   'cmake_args',
-                                   },
+                                   'cmake_args'},
```

Comment:
The rationale for the `},` on the next line is so that when appending to that list, the resulting `git diff` has a delta of just _+1 lines added_, rather than _-1+2 lines modified_. The same goes in many other places, and not just with `{}` dicts but also with `[]` lists and `()` tuples.

Putting the closing character on a separate line thus reduces the probability of merge conflicts. So you should probably undo this change, the ones at line ~1954 and ~1976, and perhaps anywhere else you might have done that.